driver/src/contracts/stablex_contract.rs

```diff
@@ -193,15 +173,15 @@ fn encode_prices_for_contract(price_vector: Vec<u128>) -> (Vec<U128>, Vec<U128>)
 fn encode_execution_for_contract(
     orders: Vec<Order>,
     executed_buy_amounts: Vec<u128>,
-) -> (Vec<H160>, Vec<U128>, Vec<U128>) {
+) -> (Vec<H160>, Vec<u64>, Vec<U128>) {
```

Comment:
Why no longer using `U128`?

Reply:
Because order**Indices** are `u16`s and not `u128`. After gnosis/ethcontract#8 it will be a `u16`.